From e04e6cf75d96ae61d52775e6721e17abae1abc8f Mon Sep 17 00:00:00 2001
From: blacksky3 <blacksky3@tuta.io>
Date: Sat, 7 May 2022 14:05:05 -0400
Subject: [PATCH] remove git version

---
 scripts/setlocalversion | 38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index af4754a35..9f5fbf345 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -44,28 +44,28 @@ scm_version()
 	fi
 
 	# Check for git and a git repo.
-	if test -z "$(git rev-parse --show-cdup 2>/dev/null)" &&
-	   head=$(git rev-parse --verify HEAD 2>/dev/null); then
+	#if test -z "$(git rev-parse --show-cdup 2>/dev/null)" &&
+	   #head=$(git rev-parse --verify HEAD 2>/dev/null); then
 
 		# If we are at a tagged commit (like "v2.6.30-rc6"), we ignore
 		# it, because this version is defined in the top level Makefile.
-		if [ -z "$(git describe --exact-match 2>/dev/null)" ]; then
+		#if [ -z "$(git describe --exact-match 2>/dev/null)" ]; then
 
 			# If only the short version is requested, don't bother
 			# running further git commands
-			if $short; then
-				echo "+"
-				return
-			fi
+			#if $short; then
+				#echo "+"
+				#return
+			#fi
 			# If we are past a tagged commit (like
 			# "v2.6.30-rc5-302-g72357d5"), we pretty print it.
-			if atag="$(git describe 2>/dev/null)"; then
-				echo "$atag" | awk -F- '{printf("-%05d", $(NF-1))}'
-			fi
+			#if atag="$(git describe 2>/dev/null)"; then
+				#echo "$atag" | awk -F- '{printf("-%05d", $(NF-1))}'
+			#fi
 
 			# Add -g and exactly 12 hex chars.
-			printf '%s%s' -g "$(echo $head | cut -c1-12)"
-		fi
+			#printf '%s%s' -g "$(echo $head | cut -c1-12)"
+		#fi
 
 		# Check for uncommitted changes.
 		# This script must avoid any write attempt to the source tree,
@@ -77,13 +77,13 @@ scm_version()
 		# it fails. Note that git-diff-index does not refresh the
 		# index, so it may give misleading results. See
 		# git-update-index(1), git-diff-index(1), and git-status(1).
-		if {
-			git --no-optional-locks status -uno --porcelain 2>/dev/null ||
-			git diff-index --name-only HEAD
-		} | read dummy; then
-			printf '%s' -dirty
-		fi
-	fi
+		#if {
+			#git --no-optional-locks status -uno --porcelain 2>/dev/null ||
+			#git diff-index --name-only HEAD
+		#} | read dummy; then
+			#printf '%s' -dirty
+		#fi
+	#fi
 }
 
 collect_files()
-- 
2.36.0

